tests/conftest.py

```diff
@@ -10,4 +10,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-pytest_plugins = "pytester"
+import pytest
+
+pytest_plugins = ["pytester"]
+
+
+def pytest_configure():
+    # Create container for tests' output
+    pytest.pytester_tmp = []
+
+
+def pytest_runtest_teardown():
+    # Clear test output object list before running new tests
+    pytest.pytester_tmp.clear()
```

Comment:
Doubt this field will be used for anything else other than outputting result of internal tests. Maybe you should use the name from the reference tests? `internal_test_output`

Reply:
I've changed the variable name because it doesn't store output, actually. It does store some kind of artifacts that we use in higher level test. And it can be used not only for one-way communication in the future.

Reply:
 `List` was used on purpose, since a test file can contain multiple tests or a parameterized test

Reply:
Ok. In this project list is not sufficient.

Reply:
Today this is so, but tomorrow some functionality will be added that will require such tests

Reply:
It will require more than just changing default type, so keep it simple.